ebsco/edsapi-koha-plugin
# Autosuggest Correction

Fixes a bug in the autosuggest trigger that was causing the defaults not to set correctly when the autosuggest link was used.  As a result, no expanders would be selected.

To Test:
http://mike-koha.ebsco-gss.net/

If you need login credentials please let me know.
Search for "presbaterian"
You should get a "Did you mean presbyterian" option on the next page.
Click the Did you mean link.
Note that the default expanders and limiters are still selected.
